### DISCUSSION TITLE
Установка AC+Боты

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#298] [Q&A]
trickerer, подскажи как с нуля установить(залить) базы ботов в АС, я что-то забыл)). В инструкции указано на примере ТС. В АС в папке Bots нет файлов указанных в инструкции. Или заливка и обновление баз ботов автоматически происходит в АС с момента первого запуска authserver и worldserver?

**Comment by trickerer** [comment]
После https://github.com/trickerer/Trinity-Bots/commit/afab801126a15a6a2a2ca4f0087bdd8bd76a4e1a базы заполняются автоматически (только AC). Оригинальная проблема была здесь: #267

**Reply by trickerer** [reply]
Файлы SQL находятся в `/data/sql/base/.../` и `/data/sql/custom/.../` (если интересно)

**Reply by xBarselonx** [reply]
Ок. Спасибо
